docs/roles/project-coordinator.rst

```diff
@@ -11,23 +11,23 @@ responsibilities and role of the project coordinator.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The project coordinator is responsible for ensuring that all club projects have the resources needed to continue developing. This may include monetary resources, connections to mentors or the like and infrastructure to host the projects. 
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-< basic eligiblity requirements – what if someone is on co-op? away from
-campus? working remote? is under academic suspension (i.e. low GPA)?
+The project coordinator must maintian good academic standing as defined by RIT to remain a member of eboard.
+Due to the hands on nature of the role, the project coordinator must be available in person during club and eboard meetings. If needed, a temporary replacement my be appointed to fill the role while the project coordinator is out on coop. In that case there should be good communication between the the two people in order to facilitate a smooth transition.
 
 
 Responsibilities
 ----------------
 
-< responsibilities and routine work that someone in this position is directly
-accountable for – what do they do and what should other eboard members expect
-from this person? >
-
+* Be physically present at all eboard meetings
```

Comment:
For lists, they should be formatted [in this way](http://docutils.sourceforge.net/docs/user/rst/quickref.html) with reST:

```rst
- item no. 1

  - sub-item no. 1

  - sub-item no. 2

- item no. 2

- item no. 3
```